labcontrol/gui/static/js/plateViewer.js

```diff
@@ -254,7 +254,37 @@ PlateViewer.prototype.initialize = function (rows, cols) {
 
   // don't select the active cell, otherwise cell navigation won't work
   this.grid.setSelectionModel(new Slick.CellSelectionModel({selectActiveCell: false}));
-  this.grid.registerPlugin(new Slick.CellExternalCopyManager(pluginOptions));
+
+  $('#multiSelectCheckbox').on('change', function() {
+    // In this callback function, "this" is the m.s. checkbox's <input> element
```

Comment:
m.s. implies they already understand enough of what they're looking at to know it means multi-select, but it's a minor issue.

Reply:
Ah, that's a fair point! I can update this to be more explicit in the future if you'd like, but I figured it was ok due to the context (since this is already in a callback function for an element called `multiSelectCheckbox`).